sunpy/map/mapbase.py

```diff
@@ -2177,8 +2177,10 @@ def contour(self, level, **kwargs):
 
         Parameters
         ----------
-        level : float
-            Value along which to find contours in the array.
+        level : float, astropy.units.Quantity
```

Comment:
just giving this functionality - it would be nice if the users could also pass a percentage level of data, this would then be consistent with `draw_contours`

Reply:
Percentage relative to what? I am reluctant to do this, given the different ways one can take a percentage of the data, and also given that it's pretty easy for a user to explicitly calculate a level as a percentage.

Reply:
percentages of the maximum values of the maps - yeah I agree its pretty easy for user. I guess it was as when I originally played with this, I assumed the levels would be in percentages as `draw_contours` is

Reply:
Heh, @dstansby...  Are we going to argue about this again? =P  (xref #4206)

Reply:
I would say this is a feature request to go on top of the original PR (https://github.com/sunpy/sunpy/pull/3909), instead of in this PR.

Reply:
I concur